command_line/to_shelx.py

```diff
@@ -192,7 +192,7 @@ def to_shelx(hklin, prefix, compound="", options=None):
     unit_cell_esds = None
     cell_data = None
     if options and options.cell:
-        if options.cell.endswith(".json"):
+        if options.cell[-5:] in (".expt", ".json"):
```

Comment:
```suggestion
        if options.cell.endswith((".expt", ".json"))
```

Reply:
Nice, I did not know `endswith` can take multiple arguments